gems/pending/spec/postgres_ha_admin/failover_monitor_spec.rb

```diff
@@ -133,6 +134,46 @@
     end
   end
 
+  describe "#failover_successful" do
+    before do
+      # @connection = PG::Connection.open(:dbname => 'vmdb_test')
```

Comment:
I'm thinking this can be removed. Attempting to create the `miq_queue` table in our database will raise an error.

If someone wants to run these tests locally I would just create a database called `travis`.